src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -133,6 +148,30 @@ public void setRegion(String region) {
         save();
     }
 
+    public String getCredentialsId() {
+        return credentialsId;
+    }
+
+    @DataBoundSetter
+    public void setCredentialsId(String credentialsId) {
+        this.credentialsId = credentialsId;
```

Comment:
`fixNull`?

Reply:
I will ensure that it is "" if it is blank, I do not like the null values ;)

Reply:
better `null` than "" for consistency IMHO

Reply:
Yes, the default is null when the field is unset initially, so a setter passed the empty string from a GUI form should normalize to null. This is potentially important for JEP-201. [Background](https://jenkins.io/doc/developer/plugin-development/pipeline-integration/#handling-default-values)

Reply:
done